pkg/features/kube_features.go

```diff
@@ -913,7 +914,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	HPAContainerMetrics:                            {Default: false, PreRelease: featuregate.Alpha},
 	SizeMemoryBackedVolumes:                        {Default: true, PreRelease: featuregate.Beta},
 	ExecProbeTimeout:                               {Default: true, PreRelease: featuregate.GA}, // lock to default and remove after v1.22 based on KEP #1972 update
-	KubeletCredentialProviders:                     {Default: false, PreRelease: featuregate.Alpha},
+	KubeletCredentialProviders:                     {Default: false, PreRelease: featuregate.Beta},
```

Comment:
This should also be on by default

Reply:
I got confused with the beta api default off KEP, but it doesn't affect feature flag.  
https://github.com/kubernetes/enhancements/issues/3136

Reply:
Done